lang/ar.js

```diff
@@ -2,44 +2,46 @@
 
 export default {
 	"components.alert.close": "إغلاق التنبيه",
-	"components.breadcrumbs.breadcrumb": "فتات الخبز",
+	"components.breadcrumbs.breadcrumb": "شريط التنقل",
```

Comment:
Now means "Navigation Bar" which seems closer to what other languages translated to